src/App.js

```diff
@@ -1,206 +1,23 @@
 import React, { Component } from "react";
-import "./App.css";
-import Pluralize from "./Pluralize";
-import Calculator from "./Calculator";
-import CalculationsTable from "./CalculationsTable";
 import Footer from "./Footer";
 import Header from "./Header";
+import Home from "./Home";
+import "./App.css";
 import "../node_modules/materialize-css/dist/css/materialize.css";
 import "../node_modules/materialize-css/dist/js/materialize.js";
 
 class App extends Component {
-  constructor() {
-    super();
-    this.state = {
-      frequency: 1,
-      unit: 20,
-      duration: 20,
-      automationTime: 2400,
-      calculationsTable: [],
-      maxSpeedUp: 0.75,
-      maxSlowDown: 2.0,
-      numberOfMonths: 36
-    };
-  }
-
-  handleChangeNumber = event => {
-    this.setState({
-      [event.target.id]: Number(event.target.value)
-    });
-  };
-
-  handleSubmit = event => {
-    event.preventDefault();
-
-    const {
-      unit,
-      frequency,
-      duration,
-      automationTime,
-      maxSlowDown,
-      maxSpeedUp,
-      numberOfMonths
-    } = this.state;
-    let numberOfTimesPerMonth = frequency * unit;
-
-    let calculationTable = Calculator(
-      duration,
-      automationTime,
-      numberOfTimesPerMonth,
-      numberOfMonths,
-      maxSpeedUp,
-      maxSlowDown
-    );
-
-    this.setState({
-      calculationsTable: calculationTable
-    });
-  };
-
-  canBeSubmitted = () => {
-    const { duration, automationTime, frequency, unit } = this.state;
-
-    return frequency > 0 && unit !== "" && duration > 0 && automationTime > 0;
-  };
-
   render() {
-    const {
-      calculationsTable,
-      duration,
-      automationTime,
-      frequency,
-      unit,
-      maxSpeedUp,
-      maxSlowDown,
-      numberOfMonths
-    } = this.state;
-    let displayResult =
-      calculationsTable.length > 1 ? (
-        <CalculationsTable table={calculationsTable} />
-      ) : (
-        ""
-      );
-    const isCalculateDisabled = !this.canBeSubmitted();
-
     return (
-      <React.Fragment>
-        <Header />
+      <div>
+        <header>
```

Comment:
Should this be in the header component, or are we going to keep the semantic layout elements (header, main, footer) here?